bamboo/bootstrap-sftp.sh

```diff
@@ -4,4 +4,5 @@ echo 'user:password' | chpasswd
 
 sed -i 's/^Port .*/Port 2222/' /etc/ssh/sshd_config
 chmod 600 /etc/authorized_keys/user
+cp -Rp /data_volume/* /data/ || true
```

Comment:
This is required to copy data from the source dir to the proper location on the unit test container, as it could be coming from a different mount point depending on which compose file is being used.